src/store/user.js

```diff
@@ -0,0 +1,24 @@
+import {createSlice} from '@reduxjs/toolkit';
+
+const initialState = {
+    userName: null,
+    isLoggedIn: false,
+}
+
+const userSlice = createSlice({
+    name: 'userSlice',
+    initialState,
+    reducers: {
+        logInHandler(state, action) {
+            state.userName = action.payload;
+            state.isLoggedIn = true;
+        },
+        logOutHandler(state){
+            state.userName = null;
+            state.isLoggedIn = false;
+        }
```

Comment:
As an option you can push user name and auth state to localStorage to prevent logout on page refresh.